api/models/Notification.js

```diff
@@ -246,7 +246,8 @@ module.exports = bookshelf.Model.extend({
       'activity.reader',
       'activity.actor'
     ]})
-    .then(notifications => notifications.map(notification => notification.send()))
+    .then(notifications => Promise.map(notifications, notification => notification.send(), {concurrency: 10}))
```

Comment:
got it

On Tue, May 31, 2016 at 12:59 PM, Lawrence Wang <notifications@github.com>
wrote:

> In api/models/Notification.js
> <https://github.com/Hylozoic/hylo-node/pull/130#discussion_r65251807>:
>
> > @@ -246,7 +246,8 @@ module.exports = bookshelf.Model.extend({
> >        'activity.reader',
> >        'activity.actor'
> >      ]})
> > -    .then(notifications => notifications.map(notification => notification.send()))
> > +    .then(notifications => Promise.map(notifications, notification => notification.send(), {concurrency: 10}))
>
> I mean concurrency between two or more different queued sendUnsent jobs
> … <#m_4617799248960656106_>
> On Tue, May 31, 2016 at 12:42 PM Robbie Carlton ***@***.***> wrote: In
> api/models/Notification.js <#130 (comment)
> <https://github.com/Hylozoic/hylo-node/pull/130#discussion_r65249191>>: >
> @@ -246,7 +246,8 @@ module.exports = bookshelf.Model.extend({ >
> 'activity.reader', > 'activity.actor' > ]}) > - .then(notifications =>
> notifications.map(notification => notification.send())) > +
> .then(notifications => Promise.map(notifications, notification =>
> notification.send(), {concurrency: 10})) @levity
> <https://github.com/levity> <https://github.com/levity> - like that? or
> concurrency somewhere else? — You are receiving this because you were
> mentioned. Reply to this email directly, view it on GitHub <
> https://github.com/Hylozoic/hylo-node/pull/130/files/2d0f14e4e8cd87ff4b58706648864385b97418b1#r65249191>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AAL0gyGhPJ85gUoYX2kQ1df0hGaTMQqtks5qHI8zgaJpZM4Iq3B->
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-node/pull/130/files/2d0f14e4e8cd87ff4b58706648864385b97418b1#r65251807>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA2Y9EEXPeULDEv7wqwRfUFloOym2kdHks5qHJMugaJpZM4Iq3B->
> .
>



-- 
Robbie Carlton

719.358.3482
<robbie@authenticmanprogram.com>